datalab/datalab_session/s3_utils.py

```diff
@@ -92,7 +92,7 @@ def get_archive_url(basename: str, archive: str = settings.ARCHIVE_API) -> dict:
   query_params = {'basename_exact': basename }
 
   headers = {
-    'Authorization': f'Token {settings.ARCHIVE_API_TOKEN}'
```

Comment:
I think it's fine to have a super user archive token here. I don't think we necessarily need to set it per-user, as the UI will only be requesting images that the user has access to.

Reply:
The point of doing this is to have the users own token passed in so the permissions they have is enforced. Right now using a superuser token, anyone could manually make a datalab api request for data they don't have permissions for and get it, so its a security issue.